concepticondata/conceptlists/Sawka-2019-201.tsv

```diff
@@ -0,0 +1,202 @@
+Sawka-2019-201-94	94	hoe	thing	284	HOE
+Sawka-2019-201-95	95	hole	thing	1667	HOLE
+Sawka-2019-201-96	96	honey	thing	942	HONEY
+Sawka-2019-201-97	97	house	thing	1252	HOUSE
+Sawka-2019-201-98	98	hunger	thing	1864	HUNGER
+Sawka-2019-201-99	99	iron	thing	621	IRON
+Sawka-2019-201-100	100	knife	thing	1352	KNIFE
+Sawka-2019-201-101	101	knot	thing	760	KNOT
+Sawka-2019-201-102	102	lake	thing	624	LAKE
+Sawka-2019-201-103	103	light	thing	630	LIGHT (RADIATION)
+Sawka-2019-201-104	104	meat	thing	634	MEAT
+Sawka-2019-201-105	105	medicine	thing	1372	MEDICINE
+Sawka-2019-201-106	106	milk	thing	635	MILK
+Sawka-2019-201-107	107	month	thing	1370	MONTH
+Sawka-2019-201-108	108	moon	thing	1313	MOON
+Sawka-2019-201-109	109	mountain	thing	639	MOUNTAIN
+Sawka-2019-201-110	110	mourning - sadness	thing	3673	MOURNING
+Sawka-2019-201-111	111	name	thing	1405	NAME
+Sawka-2019-201-112	112	night	thing	1233	NIGHT
+Sawka-2019-201-113	113	path	thing	2252	PATH
+Sawka-2019-201-114	114	porridge	thing		
+Sawka-2019-201-115	115	rain	thing	658	RAIN (PRECIPITATION)
+Sawka-2019-201-116	116	river	thing	666	RIVER
+Sawka-2019-201-117	117	root	thing	670	ROOT
+Sawka-2019-201-118	118	rope	thing	1218	ROPE
+Sawka-2019-201-119	119	salt	thing	1274	SALT
+Sawka-2019-201-120	120	sand	thing	671	SAND
+Sawka-2019-201-121	121	seed	thing	714	SEED
+Sawka-2019-201-122	122	shade	thing	1388	SHADE
+Sawka-2019-201-123	123	sickness	thing	1986	DISEASE
+Sawka-2019-201-124	124	sky	thing	1732	SKY
+Sawka-2019-201-125	125	smoke	thing	778	SMOKE (EXHAUST)
+Sawka-2019-201-126	126	spear	thing	945	SPEAR
+Sawka-2019-201-127	127	star	thing	1430	STAR
+Sawka-2019-201-128	128	stone	thing	857	STONE
+Sawka-2019-201-129	129	stool	thing	2058	STOOL
+Sawka-2019-201-130	130	sun	thing	1343	SUN
+Sawka-2019-201-131	131	thing	thing	738	THING
+Sawka-2019-201-132	132	thirst	thing	848	THIRST
+Sawka-2019-201-133	133	thorn	thing	124	THORN
+Sawka-2019-201-134	134	thread	thing	1161	THREAD
+Sawka-2019-201-135	135	today	thing	1283	TODAY
+Sawka-2019-201-136	136	tomorrow	thing	1329	TOMORROW
+Sawka-2019-201-137	137	utensil	thing		
+Sawka-2019-201-138	138	village	thing	930	VILLAGE
+Sawka-2019-201-139	139	walking stick	thing	1296	WALKING STICK
+Sawka-2019-201-140	140	wall	thing	933	WALL (OF HOUSE)
+Sawka-2019-201-141	141	war	thing	935	WAR
+Sawka-2019-201-142	142	water	thing	948	WATER
+Sawka-2019-201-143	143	what	thing	1236	WHAT
+Sawka-2019-201-144	144	wind [noun]	thing	960	WIND
+Sawka-2019-201-145	145	year	thing	1226	YEAR
+Sawka-2019-201-146	146	yesterday	thing	1174	YESTERDAY
+Sawka-2019-201-147	147	big knife	thing & adjective	1352	KNIFE
+Sawka-2019-201-148	148	branch	thing	3223	BRANCH OR TWIG
+Sawka-2019-201-149	149	leaf	thing	628	LEAF
+Sawka-2019-201-150	150	tree	thing	906	TREE
+Sawka-2019-201-151	151	sit	verb	1416	SIT
+Sawka-2019-201-152	152	brother	person	1262	BROTHER
+Sawka-2019-201-153	153	brother - younger		1760	YOUNGER BROTHER
+Sawka-2019-201-154	154	brother - older		1759	OLDER BROTHER
+Sawka-2019-201-155	155	chief	person	301	CHIEFTAIN
```

Comment:
2394?

Reply:
I felt like "The leader/ruler of a tribe/clan" would be a better fit than "The head of a company or group" because the groups questioned are tribes organized in chiefdoms. In addition, chief is very often mapped to CHIEFTAIN in other datasets already in Concepticon.